commands/general/dev.js

```diff
@@ -33,15 +33,21 @@ exports.run = async (handler, message, args, pre) => {
 
 exports.yargsOpts = {
 	alias: {
-		github: ['g', 'url', 'u'],
 		help: ['h']
 	},
-	string: ['github'],
-	required: ['github']
+	positional: {
+		args: [
+			{
+				name: 'GitHub URL',
+				type: 'string'
+			}
+		],
+		required: 1
+	}
 }
 
 exports.help = {
-	name: ['wizard', 'wiz', 'developer', 'dev'],
```

Comment:
We can leave the developer alias in. Just make dev the first item in the array so it shows as the primary name.

Reply:
Is there even a point to an alias that's literally just a longer word? Who would use that?

Reply:
Someone that doesn't know it's called `dev`, doesn't think to do `help` and guesses. There's no actual downside to keeping it in. I'd prefer, in fact, if commands had lots of related aliases that correspond to guesses a user might make; that's why I had all these four originally.